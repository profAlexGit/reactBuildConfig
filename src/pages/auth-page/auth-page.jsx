import React from 'react';

export function AuthPage () {
  return (
    <main>
      <h1>Auth page</h1>
    </main>
  );
}

export default AuthPage;
