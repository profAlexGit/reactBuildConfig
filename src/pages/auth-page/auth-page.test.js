import React from 'react';
import { configure } from 'enzyme';
import ShallowRenderer from 'react-test-renderer/shallow';
import Adapter from 'enzyme-adapter-react-16';

import { AuthPage } from './auth-page.jsx';
configure({ adapter: new Adapter() });

describe('<AuthPage />', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(<AuthPage />);
    expect(tree).toMatchSnapshot();
  });
});
