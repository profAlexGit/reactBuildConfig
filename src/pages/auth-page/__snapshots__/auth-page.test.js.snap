// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthPage /> renders correctly 1`] = `
<main>
  <h1>
    Auth page
  </h1>
</main>
`;
